### DISCUSSION TITLE
Mint problems compiling and compatibility isues with termintor

### DISCUSSION THREAD

**Discussion by ryedunn** [aristocratos/btop#67] [General]
Using mint, I could not get this to compile using gcc-10 and g++-10 (as mentioned in the documentation), without receiving the following error: 
_`g++: error: unrecognized command line option ‘-std=c++20’; did you mean ‘-std=c++2a’?`_

As of Oct-21 gcc-11 has not been included in the repositories so I needed to use the following:
```
sudo apt install build-essential manpages-dev software-properties-common
sudo add-apt-repository ppa:ubuntu-toolchain-r/test
sudo apt update && sudo apt install gcc-11 g++-11
```
After this, make was successful

Next issue is that if your font size is small enough in your emulator, the cpu cores are set to a single column which looks... well.. not as good as other emulators that split the cores into multiple columns (IMHO).  Potential feature request?

![image](https://user-images.githubusercontent.com/85773144/135765212-c739e285-5a6e-4c6a-8a81-3a5d41b67bf3.png)

**Comment by toneboy** [comment]
I have the same error, however I'm using MX Linux debian 10 (buster), so I don't think I can update the compiler. Is there a way I can depreciate, or modify so that it is backwards compatible?
Thanks!
